references/sdk.md

```diff
@@ -210,14 +227,32 @@ Create new traintuple asset.
 If `exist_ok` is true, `AlreadyExists` exceptions will be ignored and the
 existing asset will be returned.
 
+## add_aggregatetuple
+```python
+Client.add_aggregatetuple(self, data, timeout=False, exist_ok=False)
+```
+Create new aggregatetuple asset.
+`data` is a dict object with the following schema:
+```
+{
+    "algo_key": str,
```

Comment:
Same remark as for the cli regarding using `aggregate_algo_key` instead of `algo_key`